modules/apps/portal-workflow/portal-workflow-kaleo-test/src/testIntegration/java/com/liferay/portal/workflow/kaleo/internal/runtime/integration/test/WorkflowInstanceManagerImplTest.java

```diff
@@ -36,6 +50,74 @@
 public class WorkflowInstanceManagerImplTest
 	extends BaseWorkflowManagerTestCase {
 
+	@Test
+	public void testCompleteKaleoInstanceOnFailure() throws Exception {
+		String workflowDefinitionName =
+			"Broken Scripted Assignment Workflow Definition";
```

Comment:
@pedro-oliveira446 can we make it random?

Reply:
Done!